src/examples/DrawingTool/DrawingTool.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import {
+  AzureMap,
+  AzureMapDataSourceProvider,
+  AzureMapDrawingManagerProvider,
```

Comment:
we are also trying to implement the same functionality but we couldn't find this component 'AzureMapDrawingManagerProvider' could you please help us and share some infomation

Reply:
This is compatible only with https://github.com/WiredSolutions/react-azure-maps/pull/75 so you need to check out from `drawing-manager` branch and build it locally then inject to playground to have `AzureMapDrawingManagerProvider`